pkg/epp/common/config/configloader_test.go

```diff
@@ -50,21 +50,21 @@ func TestLoadConfiguration(t *testing.T) {
 		Plugins: []configapi.PluginSpec{
 			{
 				Name:       "test1",
-				PluginName: test1Name,
+				PluginName: test1Type,
 				Parameters: json.RawMessage("{\"threshold\":10}"),
 			},
 			{
 				Name:       "profileHandler",
 				PluginName: "test-profile-handler",
 			},
 			{
-				Name:       test2Name,
-				PluginName: test2Name,
+				Name:       test2Type,
```

Comment:
what about changing PluginName > Type in the configuration file and the loader?
follow up?

Reply:
In a follow up. 
Noted in PR description

Reply:
Just saw this, we can do it in a follow up